docs/quick-start.md

```diff
@@ -98,10 +98,9 @@ Congratulations! You have just made your first Amadeus for Developers API call!
 
 ## Video Tutorial
 
-You can see the process step to step in this video tutorial.
-
-![type:video](https://www.youtube.com/embed/o3SqVZJTqd8)
+You can see the process step to step in this video "How to make your first API call" from [Get Started series](https://youtube.com/playlist?list=PLBehidtj-OiqQ0sIHBPvwf-8GAjMTJehF). 
```

Comment:
- step-by-step
- I would add both link at  the video and the playlist and I wouldn't add the quotes: You can check the step-by-step process in the video tutorial of how to [make your first API call](https://www.youtube.com/watch?v=Lyy26SlgBZU&list=PLBehidtj-OiqQ0sIHBPvwf-8GAjMTJehF&index=2&t=228s) from the [Get Started with Amadeus APIs](https://youtube.com/playlist?list=PLBehidtj-OiqQ0sIHBPvwf-8GAjMTJehF) series.